nsd/check.py

```diff
@@ -0,0 +1,40 @@
+# stdlib
+import os
+import re
+
+# project
+from checks import AgentCheck
+from config import _is_affirmative
+from util import Platform
+from utils.subprocess_output import get_subprocess_output
+
+class NsdCheck(AgentCheck):
+    # Stats info https://www.nlnetlabs.nl/projects/nsd/nsd-control.8.html
+
+    NSD_CTL = ['nsd-control', 'stats']
```

Comment:
We should allow the customer to overwrite the path to `nsd-control` with an optional key in the configuration. This would be the default value.